admin/src/js/services/select2-search.js

```diff
@@ -114,26 +127,72 @@ angular.module('inboxServices').factory('Select2Search',
           });
       };
 
-      const resolveInitialValue = function(selectEl, initialValue) {
+      const addValue = (val) => {
+        if (!selectEl.children('option[value="' + val + '"]').length) {
+          selectEl.append($('<option value="' + val + '"/>'));
+        }
+      };
+
+      const getDocs = function (ids) {
+        const docsFetched = 10;
+
+        return DB()
+          .allDocs({ keys: ids, include_docs: true, limit: docsFetched })
+          .then((docs) => processDocs(docs));
+      };
+
+      const processDocs = function(result) {
+        const docs = result.rows.map(row => row.doc);
+        const docIds = docs.map(doc => doc._id);
```

Comment:
I think you can do this in a single line: 
```suggestion
const docIds = result.rows.map(row => row.doc._id);
```